### PR TITLE
Converting docstrings to numpydoc

### PR DESCRIPTION
**Pull request by yuvallanger**
Some work done on issue #466.

There are some cases I am not sure of.

I'll squash more commits to this pull request later.

### CODE REVIEW

File: glue/clients/histogram_client.py [hunk 1]
```diff
@@ -278,8 +281,7 @@ def component(self):
         return self._component
 
     def set_component(self, component):
-        """
-        Redefine which component gets plotted
+        """ Redefine which component gets plotted
```
Comment (astrofrog) [inline glue/clients/histogram_client.py]: I would prefer to leave this as it was before, i.e.

```
"""
Description.
...
```

This is what most other package that I know of do. Make sure you include a full stop at the end of the description line.

File: glue/clients/image_client.py [hunk 2]
```diff
@@ -546,17 +554,18 @@ def _get_plot_attributes(self):
...
     def _pixel_coords(self, x, y):
-        """From a slice coordinate (x,y), return the full (possibly
+        """ From a slice coordinate (x,y), return the full (possibly
         >2D) numpy index into the full data
 
-        *Note*
-        The inputs to this function are the reverse of numpy convention
-        (horizontal axis first, then vertical)
+        .. note:: The inputs to this function are the reverse of numpy
+        convention (horizontal axis first, then vertical)
```
Comment (astrofrog) [inline glue/clients/image_client.py]: The indent of the second line should match the first ``The``, so:

```
.. note:: The...
          convention...
```

**Comment by astrofrog** [comment]
@yuvallanger - this is great so far, just a few comments above. Do you plan to work more on this? (it would be great if you did!)

**Comment by yuvallanger** [comment]
I'll get back to that. What's top priority?

**Comment by astrofrog** [comment]
@yuvallanger - I don't think there are any areas that are specifically higher priority, but it would be good to do it consistently, i.e. all the docstrings in a set of files, in a sub-package, or similar. Maybe we can aim to do all the ones in ``glue.core`` and ``glue.clients`` first (since you started that here).

**Comment by astrofrog** [comment]
@yuvallanger - I merged this in https://github.com/glue-viz/glue/pull/832 (I preserved your commit). Thanks for your contribution!

**Pull request closed without merging**